### ISSUE TITLE
New license request: Parity-6.0.0 [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by kemitchell** [spdx/license-list-XML#805]
**1.** License Name: The Parity Public License 6.0.0
**2.** Short identifier: Parity-6.0.0
**3.** License Author or steward: Artless Devices LLC
**4.** URL: https://paritylicense.com/versions/6.0.0.html
**4.** OSI Status: Not Submitted

**Comment by kemitchell** [comment]
@jilayne, I included some comments in the SPDX-Tools form when submitting, but don’t see them here. Did I screw up? Should I try to reconstruct those notes from memory, in an issue comment?

**Comment by goneall** [comment]
@kemitchell I'll take a look at the online app and see what's the issue is.

**Comment by goneall** [comment]
The notes are going into the notes section of the XML and not in the issues list itself.  I've added [spdx-tools issue #98](https://github.com/spdx/spdx-online-tools/issues/98) to track.

As a workaround, you can go to the submitted licenses page, click on the license you submitted to retrieve the notes.

**Comment by goneall** [comment]
Here are the notes submitted by @kemitchell 

The Parity Public License 6.0.0 is a very strong copyleft or reciprocal public software license. Its terms were professionally drafted and further developed with extensive legal and programmer input.

A spiritual predecessor to Parity, the License Zero Reciprocal Public License, was submitted to OSI for approval, but later effectively withdrawn.

Suffice it to say that Parity remains highly controversial, under various rubrics of "free software", "open source", and standing on its own. However, nomenclature and adjudicative process aside, Parity is a public license for collaborative software development and source code distribution, a descendant of the QPL-1.0, RPL-1.1, and RPL-1.5 line, and far less restrictive than CC-BY-NC-* and CC-BY-NC-ND-*.

This license has been adopted by a diversity of licensors.  Many of those licensors are particularly concerned about the lack of an SPDX identifier, to ensure that those using automated tools can correctly identify Parity-licensed projects as an exception to the rule of permissive-dominant licensing in a variety of language and packaging communities, including npm, Maven, Cargo, and RubyGems.

Three public projects kindly gave permission to be listed in support of this application:

https://freckles.io/#licensing

http://scion.scxml.io/

https://github.com/agreements-network/blackstone

Some projects use Parity as the public license part of a commercial dual-licensing business model. Others, like Monax, developer of Blackstone, use it simply for its strong reciprocal effect.  Artless Devices, the license steward, uses the license in the latter way, as well.

There are two template fields, notated with [square brackets] in the provided plain-text rendering.

**Comment by kemitchell** [comment]
Thanks, @goneall! Long time, no @-mention! :wave:

**Comment by goneall** [comment]
@kemitchell @jlovejoy FYI - The spdx-online app has been updated to properly handle the comments.

**Comment by tunnckoCore** [comment]
Great, I'm probably not using it because of this reason too. And will wait for SPDX ID for the Prosperity-2.0.0 too.

**Comment by jlovejoy** [comment]
@tunnckoCore - FYI, I don't see Prosperity 2.0.0 as submitted to be added. Someone has to submit it first, then the SPDX legal team has to review it, and then there is a fair amount of work to actually get the license on the list in the right format for the next SPDX License List release. See: https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md
There is a very small number of volunteers who do this work. We'd be happy to have more help and feedback on how to communicate this reality more broadly! (preferably, via our mailing list)
thanks.

**Comment by kemitchell** [comment]
@jlovejoy Would it help the team if I took the time to format this request as a pull request, with all the XML and other elements ready to go?

**Comment by jlovejoy** [comment]
@kemitchell - that would be very helpful, but only after (and if) the license is accepted to be added. no sense in doing the work until that point.

**Comment by tunnckoCore** [comment]
@jlovejoy yup, I was just mentioning it.

**Comment by swinslow** [comment]
Sharing my gut reaction thoughts, upon reading the license text:

I admit I'm having some qualms in sorting through whether I think this falls within the current [License Inclusion Principles](https://spdx.org/spdx-license-list/license-list-overview).

Agree with the comments above that it's less restrictive than some other licenses that are currently on the license list, as it doesn't contain a noncommercial-only restriction like CC-BY-NC-*. (I also understand that the license inclusion principles have evolved over time, and that some licenses that were previously added might not now be added if they were being presented for the first time.)

I do think there are some interesting questions here that are worth the Legal Team discussing, particularly around clause #3 (and at a high level, I gather that clause is one of the main distinctions between this license and many others). I'd suggest that the Legal Team should discuss this one in the context of other recent considerations around the scope of the license inclusion guidelines, in one of the calls during the current 3.6 release cycle.

**Comment by kemitchell** [comment]
@swinslow, I'd be happy to join that call.

**Comment by swinslow** [comment]
Discussed initially on 2019-04-18 Legal Team call. The Legal Team is continuing to discuss but did not yet reach consensus, and expects to raise again on a subsequent call.

**Comment by swinslow** [comment]
Circling back on this one:

Based on the updated [License Inclusion Principles](https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md), and recent conversations on the spdx-legal mailing list, I'm more inclined to say that this is ok to include on the list. 

I do think it's a close call, particularly given the mix of (1) the scope and effect of clause 3, and (2) I'm not sure how well it meets the "broad usage in the wild" considerations. Of the three projects mentioned above, from a super-superficial look, they didn't appear to be in wide use, and v6 of the Parity license text only appears to be used by two of them. (For SCION, looks like they might be using v3? see, e.g., https://scion.scxml.io/blog/scion-5-release-announcement/#license-evolution, https://gitlab.com/scion-scxml/debug)

That said, at the moment, I'm (personally) tilting towards including it on the list. I do think it's probably appropriate for the broader SPDX Legal Team to weigh in, and I plan to bring it up on the next call (2019-06-13).

**Comment by kemitchell** [comment]
@swinslow should I plan to join that call?

**Comment by swinslow** [comment]
@kemitchell up to you, you (and others) are always welcome to join!  =)  There may also be additional feedback from reviewers / participants in this thread in the meantime.

**Comment by jlovejoy** [comment]
+1 to add to list

**Comment by swinslow** [comment]
Discussed on 2019-06-13 legal team call, agreed to add.

**Comment by tunnckoCore** [comment]
@swinslow oooooooh yea! :))

**Comment by tunnckoCore** [comment]
When we'll be able to see it online and to use it? (like in npm license field?)

**Comment by kemitchell** [comment]
@tunnckoCore as soon as the SPDX team updates the SPDX website, @shinnn and I can update https://github.com/shinnn/spdx-license-ids. I'm on the lookout, and will do it as soon as I can.

**Comment by swinslow** [comment]
The license list website at https://spdx.org/licenses is updated on a quarterly basis with each new release of the license list. The next release (3.6) will be coming up at the end of this month, or shortly thereafter.

SPDX's [license-list-data repo](https://github.com/spdx/license-list-data) will be tagged and updated for the 3.6 release at the same time.

Thereafter, I can't speak to when NPM or any other community will update their own license list copies to the new version, as that's up to them -- but it sounds like @kemitchell is on top of that  =)

**Comment by tunnckoCore** [comment]
Cool, thanks. It's probably when the ids package is released.

**Comment by shinnn** [comment]
I, one of the maintainers of `spdx-license-ids` npm package, can change its release policy from keeping the list in sync with the website to doing it with `license-list-data` repo.

However, I doubt that the latter is better for end users; it might be confusing if a user find a license in `spdx-license-ids`’s list which doesn’t appear in the website.

> The license list website at https://spdx.org/licenses is updated on a quarterly basis with each new release of the license list.

Could the website be more up-to-date by automating its release by using a CI service?

**Comment by zvr** [comment]
As @swinslow wrote, *releases* happen on quarterly basis and are tagged -- and the list, with a new version number, is published on the website.

The repositories HEAD (for this repository and license-list-data) is work-in-progress and should not be used externally -- or at least, without expecting that things may change, that content is unusable, etc.

**Comment by goneall** [comment]
As @zvr you can use the tags on the license-list-data to get the latest stable, but it will be exactly the same data you are pulling from the website since the tools that generate license-list-data also generate the pages for the website.

For [SPDX tools](https://github.com/spdx/tools), we use the website as the primary source fetching the latest license list data and fall back to a local cache of licenses if internet connectivity is not available.  The local cache is updated when new releases of the SPDX tools are made available.

**Comment by shinnn** [comment]
@zvr @goneall Thanks for the clarification. I think I had misread https://github.com/spdx/license-list-XML/issues/805#issuecomment-505150415 a bit.

**Comment by feross** [comment]
I still don't see the Prosperity license listed on the website. Is this still pending?

**Comment by feross** [comment]
Here's a PR to add Prosperity-3.0.0: https://github.com/spdx/license-list-XML/pull/960
